src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -246,15 +246,18 @@ def arguments_loader():
     def handler(args):  # pylint: disable=too-many-branches,too-many-statements
         from msrestazure.azure_operation import AzureOperationPoller
 
-        ordered_arguments = args.pop('ordered_arguments') if 'ordered_arguments' in args else []
+        ordered_arguments = args.pop('ordered_arguments', [])
+        for item in ['properties_to_add', 'properties_to_set', 'properties_to_remove']:
+            if args[item]:
+                raise CLIError("Unexpected '{}' was not empty.".format(item))
```

Comment:
There's an Action somewhere that removes the entries in these lists and puts them into the "ordered_args" argument. So if anything was left in these, I want to raise an error so I know there's a bug in the generic update logic before I delete them.